src/js/utils/ApplicationUtil.js

```diff
@@ -0,0 +1,60 @@
+import {
+  MESOS_SUMMARY_CHANGE,
+  MESOS_SUMMARY_REQUEST_ERROR
+} from '../constants/EventTypes';
+import Config from '../config/Config';
+import MesosSummaryStore from '../stores/MesosSummaryStore';
+
+module.exports = {
+  /**
+   * Checks that enough time has elapsed before rendering app
+   *
+   * @param  {Func} onDelayEndCallback Gets called after delay has ellapsed
+   */
+  renderOnDelayEnd(onDelayEndCallback) {
```

Comment:
Did the first part, the second part I don't think we need just now, so I didn't do it.